docs/create_rules.md

```diff
@@ -90,3 +90,60 @@ def sql_has_reasonable_number_of_lines(model: Model, max_lines: int = 200) -> Ru
             message=f"SQL query too long: {count_lines} lines (> {max_lines})."
         )
 ```
+
+### Filtering
```

Comment:
There's many ways to filter! I would say we try to nudge people into using one way. IMO that would be using the `pyproject.toml` file. So people would create a `@model_filter` and apply it to a rule in `pypoject.toml`. Or do you think using `SkipRule()` and `@rule(model_filters={only_schema_x()})` is more straightforward?

My main concern is that, depending on if you own the rule or not, the way of filtering changes

Reply:
There are three ways of filtering: two are shared with Severity (`pyproject.toml` and inside `@rule`). The other is the feature of a custom rule returning `SkipRule()`.

If we are going to change it, I'd say dropping the latter is better, so we keep parity with Severity. However I personally like this feature, it may feel cumbersome to create a `ModelFilter` that is only going to be used once by a custom rule.

Maybe I just wrote the documentation too dense, and it can be seen as a separate thing. Or maybe it is in fact too much and we should drop it. I'll let you decide on this.

Reply:
If you like the feature, I think we should not drop it! I like the idea of documenting it as a separate thing :+1: